modules/sff/module/python/sff/enums.py

```diff
@@ -213,7 +213,10 @@ class SFF_MODULE_TYPE(Enumeration):
     _1G_BASE_T = 49
     _100_BASE_LX = 50
     _100_BASE_FX = 51
-    _4X_MUX = 52
+    _100G_BASE_LR = 52
```

Comment:
You should group the 100G enums together.

Reply:
grouped 100G...